server/src/routes/follow.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const Follows = require('../db/mongo/models/follow');
+
+const router = express.Router();
+
+router.get('/:email_from/followers', async (req, res) => {
+    const emailFrom  = req.params.email_from;
+    try {
+        let followers = await Follows.find({ email_from: emailFrom}).exec();
+        res.send(followers);
+        console.log(`Sending requested followers of ${emailFrom}`);
+    } catch (e) {
```

Comment:
Use `response` and `statusCode` variables for handling try/catch scenarios.
Refer to mine's / Shuker's branches that do the same.
Relevant to every route in this file